composer.json

```diff
@@ -4,11 +4,15 @@
     "type": "project",
     "description": "Symfony Demo Application",
     "autoload": {
-        "psr-4": { "": "src/" },
+        "psr-4": {
```

Comment:
@theofidry the symfony-demo project has a AppBundle namespace for its source, because this is what the Symfony best practices are telling (and the demo is also about showing these best practices).

The extra CodeExplorerBundle is more a project-specific vendor bundle actually (we haven't shared it as a real vendor bundle, because this feature simply does not make sense outside the demo project).

and enw users won't find it harder, as they won't need to change the autoload config of symfony-demo.